src/test/resources/org/jenkinsci/plugins/github_branch_source/github-app-jcasc-minimal-expected-export.yaml

```diff
@@ -6,6 +10,9 @@ system:
         description: "GitHub app 1111"
         id: "github-app"
         privateKey: "some-secret-value"
+        repositoryAccessStrategy:
+          specificRepositories: {
+            }
```

Comment:
Note that all of _these_ cases of empty objects reflect design mistakes in some plugin or another: the configuration code should be fixed to default to null fields. https://www.jenkins.io/doc/developer/plugin-development/pipeline-integration/#handling-default-values is about Pipeline steps but the same principles apply. So if and when those mistakes are fixed, this test would be adjusted correspondingly.

The motivation to skip empty objects in the original JCasC implementation was not explained, but presumably it was an attempt to produce simpler exports by working around these plugin mistakes. Yet it prevented plugins from _deliberately_ treating the presence or absence of an object (typically `f:optionalProperty` in the GUI) as significant.